BCH-Wunder-Project-team-4/Wunder.io
# Content

## Link to ticket:

<img width="611" alt="Screenshot 2023-12-19 at 13 34 01" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/6374c807-c8a3-4492-a06e-a142af69564e">


## Link to feature environment:
You can test this page ´:
http://localhost:3000/all-articles/revolutionizing-decoupled-development-open-source-blueprint-drupal-and-nextjs-projects

## Changes proposed in this PR:

added on more line to gitignore for sharing inline-images

## How to test:

1.  probably you have to delete this folder on your machine
2. git fetch origin content
3. git checkout content


That should be it.. 
if not try lando drush cim
